client/api/notebook.py

```diff
@@ -79,6 +95,8 @@ def save_notebook(self):
         # Wait for first .ipynb to save
         if ipynbs:
             if wait_for_save(ipynbs[0]):
+                # Optional delay to increase likelihood of complete save.
+                time.sleep(delay)
```

Comment:
Is this delay is necessary? The `save` function will sleep those extra 0.5 seconds anyway if the file is corrupted.